salt/profiles/config/srv-profiles-app.cfg

```diff
@@ -1,3 +1,14 @@
-base_url: {{ pillar.profiles.base_url }}
-orcid_client_id: {{ pillar.profiles.orcid.client_id }}
-orcid_client_secret: {{ pillar.profiles.orcid.client_secret }}
+[oauth2]
+    [[clients]]
+        {% for name, details in pillar.profiles.get('oauth_clients', {}).items() %}
+        [[[{{ name}}]]]
```

Comment:
is `[[[` rather than `[[` intentional?

Reply:
Yep, third level. (Maybe top two levels could be merged.)